config/puma.rb

```diff
@@ -5,3 +5,9 @@
 workers Integer(ENV.fetch('PUMA_WORKERS', 0))
 threads_count = Integer(ENV.fetch('PUMA_THREADS', 16))
 threads(threads_count, threads_count)
+
+preload_app!
+
+on_worker_boot do
+    ActiveRecord::Base.establish_connection
```

Comment:
Style nitpick:  are there too many spaces here?

Reply:
Right you are